capstone/src/main/resources/static/event.js

```diff
@@ -1,25 +1,72 @@
+/* Function to prefill event information if editing event */
+function loadEventInfo() {
+  const event = window.location.hash.substring(1);
+  if (event != "") {
+    fetch('get-event?event-id=' + event).then(response => response.json()).then((data) => {
+      document.getElementById("eventTitle").value = data.eventTitle;
+      document.getElementById("eventDateTime").value = data.eventDateTime;
+      document.getElementById("eventLatitude").value = data.eventLatitude;
+      document.getElementById("eventLongitude").value = data.eventLongitude;
+      document.getElementById("eventDescription").value = data.description;
+      document.getElementById("event-id").value = data.datastoreId;
+      if (data.foodAvailable == true) {
+        document.getElementById("foodAvailable").checked = true;
+      }
+      if (data.requiredFee == true) {
+        document.getElementById("requiredFee").checked = true;
+      }
+    });
+  }
+}
+
+/* Function to create Google Map */
+async function createMap() {
+  var princetonLatLng = {lat: 40.3428452, lng: -74.6568153};
+  const campusMap = new google.maps.Map(
+    document.getElementById('map'),
+    {center: princetonLatLng, zoom: 16});
+
+   const response = await fetch('get-all-events');
+   const jsonEvents = await response.json();
+   jsonEvents.forEach(event => createMarker(event, campusMap));
+}
+
+/* Create a new marker for each event
+ * @param event - event object
+ * @param campusMap - Google Map object
+ */
+function createMarker(event,campusMap) {
+  var eventPosition = {lat: event.eventLatitude, lng: event.eventLongitude};
+  const newMarker = new google.maps.Marker({
+    map: campusMap,
+    title: event.eventTitle,
+    position: eventPosition
+  })
+}
```

Comment:
This might have dropped in from another PR.

Reply:
Oh I just moved some of the functions around so they'd be grouped by functionality!